keybot/keybot.go

```diff
@@ -84,8 +84,8 @@ func (k *keybot) Run(bot slackbot.Bot, channel string, args []string) (string, e
 	case buildAndroid.FullCommand():
 		skipCI := *buildAndroidSkipCI
 		automated := *buildAndroidAutomated
```

Comment:
Since you're in here let's make automated skip ci. Given how busted ci is constantly we never get this build out

Reply:
Think that is better done in the command that is run on a timer.